inscriptions/ISic001401.xml

```diff
@@ -190,19 +195,18 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="PHI" n="PHI140909">
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#AL">
                 <ab>
-                    <lb n="1"/>
-
-
-                    Βετία Κο<supplied reason="omitted">ρν</supplied>η-<lb n="2"/>λία εὐσεβεστ<supplied reason="omitted">ά</supplied>-<lb n="3"/>τ<supplied reason="omitted">η</supplied>, ἔζησε ∙ ἔτ
-                    <supplied reason="omitted">η</supplied>
-                    <lb n="4"/> νʹ.
-                    <lb n="5"/>
+                    <lb n="1"/><persName type="attested"><name type="nomen">Βετία</name> <space quantity="1" unit="character"/> <name type="cognomen" cert="low">Κο<choice><corr>ρν</corr><sic>μ</sic></choice>η
```

Comment:
again, both here in line 1, and the final letter of line 3, I think one simply reads ρν and η and then reports the reading of Ansaldi in the apparatus, since it seems infinitely more likely that he found it difficult to read and report what was visible, than that this is what was really originally engraved.